src/scoped.component.test.js

```diff
@@ -61,6 +61,93 @@ describe("<Scoped />", function() {
     ReactDOM.unmountComponentAtNode(el);
   });
 
+  it("should create local CSS for combined CSS statements", function() {
```

Comment:
the names for all of these tests is the same which could be confusing when one fails.